config/application.yml.example

```diff
@@ -200,7 +202,9 @@ production:
   session_encryption_key: # generate via `rake secret`
   session_timeout_in_minutes: '8'
   state_id_proofing_vendor: 'mock'
-  twilio_accounts: # '[{"sid":"sid", "auth_token":"token", "number":"9999999999"}]'
+  twilio_numbers: '["9999999999","2222222222"]'
+  twilio_sid: 'sid1'
+  twilio_auth_token: 'token1'
```

Comment:
It seems dangerous to have these defaults under `production`. If you deploy this change to a production like AWS environment without changing the `application.yml` in app-secrets to use the new config format, you won't find out about your mistake until runtime when the new worker servers attempt to make Twilio requests on live traffic with this bogus Twilio auth token.

This is because in production the `application.yml.example` values are used as defaults, and the secrets in S3 are overlaid to get the final values.

I would suggest not setting any of these under `production`, or else checking in the app and raising at start-up time that these need to be set. This would be safer because it would be obvious at server build time that the service was not healthy.

Reply:
This is what the errors currently look like — not very helpful if you don't know what you're looking for.

```
W, [2018-01-25T18:52:25.797962 #2169]  WARN -- : {"context"=>"Job raised exception", "job"=>{"class"=>"ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper", "wrapped"=>"SmsOtpSen
derJob", "queue"=>"sms", "args"=>[{"job_class"=>"SmsOtpSenderJob", "job_id"=>"ee09f90e-0426-4da5-921a-f342716efafb", "provider_job_id"=>nil, "queue_name"=>"sms", "priority"=>nil, 
"arguments"=>"[redacted]", "executions"=>0, "locale"=>"en"}], "retry"=>false, "jid"=>"59063c89ee39ee89afb8a894", "created_at"=>1516906311.2571785, "enqueued_at"=>1516906345.210224
4, "error_message"=>"Unable to create record: The requested resource /2010-04-01/Accounts/sid1/Messages.json was not found", "error_class"=>"Twilio::REST::RestError", "failed_at"=
>1516906311.7677152, "retry_count"=>0}, "jobstr"=>"[redacted]"}
W, [2018-01-25T18:52:25.798084 #2169]  WARN -- : Twilio::REST::RestError: Unable to create record: The requested resource /2010-04-01/Accounts/sid1/Messages.json was not found
W, [2018-01-25T18:52:25.798133 #2169]  WARN -- : /srv/idp/shared/bundle/ruby/2.3.0/gems/twilio-ruby-5.5.0/lib/twilio-ruby/framework/version.rb:157:in `create'
```